hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ec/reconstruction/ECReconstructionCoordinator.java

```diff
@@ -324,9 +324,11 @@ private void checkFailures(ECBlockOutputStream targetBlockStream,
       // Even after retries if it failed, we should declare the
       // reconstruction as failed.
       // For now, let's throw the exception.
-      throw new IOException(
-          "Chunk write failed at the new target node: " + targetBlockStream
-              .getDatanodeDetails() + ". Aborting the reconstruction process.");
+      String message = "Chunk write failed at the new target node: " +
```

Comment:
This would create redundant logs. As far as I understand this is already being logged in the class ECReconstructionCoordinatorTask.

Reply:
OK, I remove the extra log.